### PR TITLE
Adding Multi model serving

### PR DESCRIPTION
**Pull request by asheet**
added rbac rhoai projects to training readme and closed old pr

### CODE REVIEW

**Comment by asheet** [comment]
Will need to merge main into this branch after single-model-serving gets merged. Linting failing on fixes that are in single-model-serving branch.

File: tenants/ai-example/multi-model-serving/base/inference-service.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    model:
+      modelFormat:
+        name: onnx
+        version: '1'
+      name: ''
+      resources: {}
+      runtime: multi-model-server
+      storage:
+        key: aws-connection-multi-model
+        path: models/fraud-detection-model/tenants/ai-example/multi-model-serving/models/frauddetectionmodel.onnx
```
Comment (strangiato) [inline tenants/ai-example/multi-model-serving/base/inference-service.yaml:21]: How difficult would it be to update the path to be something like this with the upload job?

models/fraud-detection-model/frauddetectionmodel.onnx

**Review by strangiato — CHANGES_REQUESTED** [review]
Looks great!

I suggested a few minor updates on the connection secret.

Also, I think it would be great to simplify the path for the model if it isn't too much work.  Not a big deal to leave it as is if it isn't a simple update.

File: tenants/ai-example/multi-model-serving/base/data-connection.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    opendatahub.io/managed: 'true'
+  annotations:
+    opendatahub.io/connection-type: s3
+    openshift.io/display-name: multi-model
+stringData:
+  AWS_ACCESS_KEY_ID: minio
+  AWS_S3_BUCKET: multi-model
+  AWS_S3_ENDPOINT: http://minio.ai-example-multi-model-serving.svc.cluster.local:9000
+  AWS_SECRET_ACCESS_KEY: minio123
+  AWS_DEFAULT_REGION: us-east-1
```
[the remark this answers is not in the document]
Comment (asheet) [reply tenants/ai-example/multi-model-serving/base/data-connection.yaml]: tested, we don't need the region. removed.

File: tenants/ai-example/multi-model-serving-minio/base/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ai-example-multi-model-serving
+
+resources:
+  - ../../../../components/apps/minio/overlays/large-files
```
[the remark this answers is not in the document]
Comment (asheet) [reply tenants/ai-example/multi-model-serving-minio/base/kustomization.yaml]: changed to default

File: tenants/ai-example/multi-model-serving/base/inference-service.yaml (hunk 1 above)
Comment (asheet) [reply tenants/ai-example/multi-model-serving/base/inference-service.yaml]: updated the pathing, had to change the other examples

**Review by strangiato — APPROVED** [review]

**Pull request merged**